src/io.cpp

```diff
@@ -54,6 +54,44 @@ class ToWKT {
     std::shared_ptr<s2geog::WKTWriter> m_writer;
 };
 
+#if defined(S2GEOGRAPHY_VERSION_MAJOR) && \
+    (S2GEOGRAPHY_VERSION_MAJOR >= 1 || S2GEOGRAPHY_VERSION_MINOR >= 2)
+class FromWKB {
+public:
+    FromWKB(bool oriented, bool planar, float tessellate_tolerance = 100.0) {
+        s2geog::geoarrow::ImportOptions options;
+        options.set_oriented(oriented);
+        if (planar) {
+            auto tol = S1Angle::Radians(tessellate_tolerance / EARTH_RADIUS_METERS);
+            options.set_tessellate_tolerance(tol);
+        }
+        m_reader = std::make_shared<s2geog::WKBReader>(options);
+    }
+
+    PyObjectGeography operator()(py::bytes a) const {
+        return make_py_geography(m_reader->ReadFeature(a));
+    }
+
+private:
+    std::shared_ptr<s2geog::WKBReader> m_reader;
```

Comment:
Use `unique_ptr` instead of `shared_ptr`? (Also in `ToWKB`) Unless I'm missing something?

Reply:
It might be that pybind11 requires this class to be copyable? (if you pass it as argument to vectorize) 
And that then gives compilation errors if using unique_ptr instead of shared_ptr

Reply:
> It might be that pybind11 requires this class to be copyable? (if you pass it as argument to vectorize)

Ah yes I see. Another option may be to add a `FromWKB` copy constructor (since this is what is passed to `py::vectorize`) although using a shared pointer is certainly fine.

Reply:
I would personally leave it as is with a shared pointer then, to not make the small class more complicated (I don't think instantiation of the vectorized function is performance sensitive?)